src/migration/1687960180394-PostFlags.ts

```diff
@@ -0,0 +1,22 @@
+import { MigrationInterface, QueryRunner } from "typeorm";
+
+export class PostFlags1687960180394 implements MigrationInterface {
+    name = 'PostFlags1687960180394'
+
+    public async up(queryRunner: QueryRunner): Promise<void> {
+        await queryRunner.query(`ALTER TABLE "post" ADD "flags" jsonb NOT NULL DEFAULT '{}'`);
```

Comment:
@idoshamun should we set DEFAULT as:
```
ALTER TABLE "post" ADD "flags" jsonb NOT NULL DEFAULT '{"sentAnalyticsReport":true,"banned":false,"deleted":false,"private":false,"visible":true,"showOnFeed":true,"promoteToPublic":false}''
```

To have default values available for new created posts? It makes sense to set those since not all are `false` by default and we can not do "false if missing".

Reply:
Let's do it only for the true by default values

Reply:
#savingdbstorage works!